app/src/main/java/com/github/pockethub/ui/issue/DeleteCommentTask.java

```diff
@@ -62,7 +62,7 @@ public DeleteCommentTask(final Context context,
 
     @Override
     protected GithubComment run(Account account) throws Exception {
-        new DeleteIssueCommentClient(context, InfoUtils.createRepoInfo(repository), comment.id);
+        new DeleteIssueCommentClient(context, InfoUtils.createRepoInfo(repository), comment.id).executeSync();
```

Comment:
I feel like a idiot for the moment... need to be more awake when re-writing code